The Globe and Mail.js

```diff
@@ -1,223 +1,159 @@
-					var author = doc.evaluate('//div[@id="author"]/p[@class="byline"]', doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
-				}
-				noMetaType = 0;
-		}
-		
-		var metaTags = new Object();
-		var tagsContent = new Array();
-		
-	//get data
-		var metaTagHTML = doc.getElementsByTagName("meta");
-		for (var i = 0 ; i < metaTagHTML.length ; i++) {
-			metaTags[metaTagHTML[i].getAttribute("name")] = Zotero.Utilities.cleanTags(metaTagHTML[i].getAttribute("content"));
-		}
-		
-	//author	
-		if (metaTags["byline"]) {
-			var author = metaTags["byline"];
-		}
-		
-	//date	
-		if (metaTags["pubdate"]) {
-			var month = metaTags["pubdate"].substr(4, 2);
-			var day = metaTags["pubdate"].substr(6, 2);
-			var year = metaTags["pubdate"].substr(0, 4);
-			
-			newItem.date = (year + "-" + month + "-"+ day);
-		}
-	
-	//tags	
-		if (metaTags["article_keywords"]) {
-			tagsContent = metaTags["article_keywords"].split("; ");
-		}
-		
-		for (var i = 0; i < tagsContent.length; i++) {
-		     	if (tagsContent[i] != (" ") && tagsContent[i] != ("")) {
-			     	newItem.tags[i] = tagsContent[i];
-		     	}
-	     	}	
-		
-		associateMeta (newItem, metaTags, "headline", "title");
-		associateMeta (newItem, metaTags, "summary", "abstractNote");
-		associateMeta (newItem, metaTags, "desk", "section");	
-		associateMeta (newItem, metaTags, "article_id", "callNumber");
-		associateMeta (newItem, metaTags, "credit", "rights");
-	
-	//rest of author (shared between both newspaperArticle types)
-		if (author) {
-			
-		
-			if (author.substr(0,3).toLowerCase() == "by ") {
-				author= author.substr(3);
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
 			}
-		
-			var authors = author.toLowerCase().split(" and ");
-			for (var i=0; i<authors.length; i++) {
-				var author = authors[i];
-				var words = author.split(" ");
-				
-				for (var j=0; j<words.length; j++) {
-					if (words[j] != "") {
-						words[j] = words[j][0].toUpperCase() + words[j].substr(1).toLowerCase();
-					}
-				}
-				author = words.join(" ");
-				newItem.creators.push(Zotero.Utilities.cleanAuthor(author, "author"));			
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
 			}
-		}
-	}	
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
 	
-	newItem.url = doc.location.href;
-	newItem.complete();
+	newItem = new Zotero.Item("newspaperArticle");
+	newItem.url = url;
+
+	//get headline
+	var title = ZU.xpathText(doc, '//article/h1');
+	if (!title) title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	newItem.title = title;
+
+	//get abstract
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@property="og:description"]/@content');
 	
-}
+	//get date
+	var xpathdate_author = '//time[@itemprop="datePublished"]';
```

Comment:
Rename this variable (it has nothing to do with the author here) or maybe just write these two lines together.